src/execution/execute.ts

```diff
@@ -60,12 +60,13 @@ import {
 import { typeFromAST } from '../utilities/typeFromAST';
 import { getOperationRootType } from '../utilities/getOperationRootType';
 
+import { Maybe } from '../jsutils/Maybe';
+
```

Comment:
Can you please review commit that introduces `Maybe` and fit it there?
Note: `jsutils` imports should be in the same group and should after `node_modules` imports.